src/wranglerArtifactManager.test.ts

```diff
@@ -0,0 +1,87 @@
+import { describe, expect, it } from "vitest";
+import {
+	getDetailedPagesDeployOutput,
+	getWranglerArtifacts,
+} from "./wranglerArtifactManager";
+
+describe("wranglerArtifactsManager", () => {
+	const mock = require("mock-fs");
+
+	describe("getWranglerArtifacts()", async () => {
+		it("Returns only wrangler output files from a given directory", async () => {
+			mock({
+				testOutputDir: {
+					"wrangler-output-2024-10-17_18-48-40_463-2e6e83.json": `
+                    {"version": 1, "type":"wrangler-session", "wrangler_version":"3.81.0", "command_line_args":["what's up"], "log_file_path": "/here"}
+                    {"version": 1, "type":"pages-deploy-detailed", "environment":"production", "alias":"test.com", "deployment_id": "123", "url":"url.com"}`,
+					"not-wrangler-output.json": "test",
+				},
+			});
+
+			const artifacts = await getWranglerArtifacts("./testOutputDir");
+
+			expect(artifacts).toEqual([
+				"./testOutputDir/wrangler-output-2024-10-17_18-48-40_463-2e6e83.json",
+			]);
+			mock.restore();
```

Comment:
We could probably move `mock.restore()` in to an `afterEach()`

Reply:
Good point